front-end/src/App.js

```diff
@@ -14,6 +15,8 @@ class App extends Component {
         <div className="app">
           <Header/>
           <Login/>
+          <Forgot/>
+          {/*<Route exact path="/forgot" component="forgot"/>*/}
```

Comment:
It is not a good practice to commit commented code, unless you have a very good reason to do that. You should either remove the code completely if you no longer need it, or remove the comments and keep the code.